README.md

```diff
@@ -4,24 +4,18 @@ This repository contains mobile standards created for mobile platforms. It inclu
 
 ## Table of contents
 
-1. [Introduction](docs/general/introduction.md)
-
-2. [Accessibility features per mobile platform](docs/features/features_mobile_platforms.md)
-
-    * [Android](docs/features/features_android.md)
-    * [iOS](docs/features/features_ios.md)
-    * [Flutter](docs/features/features_flutter.md)
-
-3. [Accessibility guidelines](docs/guidelines/accessibility_guidelines.md)
-
-    * [Accessibility principles and examples](docs/guidelines/principles/accessibility_principles_and_examples.md)
-
-       * [Perceivable](docs/guidelines/principles/perceivable_principle.md)
-       * [Operable](docs/guidelines/principles/operable_principle.md)
-       * [Understandable](docs/guidelines/principles/understandable_principle.md)
-       * [Robust](docs/guidelines/principles/robust_principle.md)
-       * [Guideline checklist](docs/guidelines/principles/guideline_checklist.md)
-  
-    * [Testing accessibility features](docs/testing/testing.md)
-
-4. [Conclusion](docs/general/conclusion.md)
+* [Introduction](docs/introduction.md)
+* [Guidelines](docs/accessibility_guidelines.md)
+* [Checklist](docs/accessibility_checklist.md)
+* [Principles](docs/accessibility_principles.md)
+    * [Perceivable](docs/principles/perceivable_principle.md)
+    * [Operable](docs/principles/operable_principle.md)
+    * [Understandable](docs/principles/understandable_principle.md)
+    * [Robust](docs/principles/robust_principle.md)
+* [Testing accessibility](docs/accessibility_testing.md)
+* [Conclusion](docs/conclusion.md)
+
+* **[Platforms specifics](docs/platforms/features_mobile_platforms.md)**
+    * [Android](docs/platforms/android/features_android.md)
+    * [iOS](docs/platforms/ios/features_ios.md)
+    * [Flutter](docs/platforms/flutter/features_flutter.md)
```

Comment:
Related to ordering, I would suggest to go with number for easier navigation; e.g. when discussing with projects teams, it was easier to say; "check chapter 3.1 for this".

Reply:
I do have a concern with using numbers for chapters.
I'm afraid users will be confused because then we will have 2 numbering systems:
* **Chapters**: each chapter has a number: `1. Introduction`, `2. Guidelines`, ... `4.1 Perceivable`...
* **WCAG**: organizes guidelines by numbers: `1.1.1 Non-text content (Level A)`...

For example, the user will say: "check chapter `4.1 Perceivable`", but when he opens the chapter, he will find `1.1.1 Non-text content (Level A)`
This seems really confusing.

Reply:
I do undestand and it's totally valid, but based on the current experience, numbering was something that was easier to understand the guidelines overall.

I would say it is totally fine to have numbers like that, because those numbers (from WCAG) refers to the standard itself and it's just a reference to the standard itself. And that can be beneficial if they will need to find more about it.

On the other side, as I've mentioned, I got quite a lot of questions where to look things and it was really easy to tell to people look at chapter 2.1 or 4.2, etc.